tests/mmm/test_multidimensional.py

```diff
@@ -813,12 +813,13 @@ def create(
         prefix: str = "holiday",
         sigma_dims: str | None = None,
         effect_size: Prior | None = None,
+        dims: tuple[str] | str | None = None,
     ):
         basis = GaussianBasis()
         return EventEffect(
             basis=basis,
             effect_size=Prior("Normal"),
-            dims=(prefix,),
+            dims=dims or (prefix,),
```

Comment:
I wanted to enable specifying dimensions for the EventEffect, but kept the default behaviour unchanged.